Autoplay plugin E2E test sometimes fails randomly

Perhaps related to `t.wait` on line 12?

```
 Running tests in:
 - Chrome 83.0.4103.61 / Linux 0.0
 - Firefox 76.0 / Linux 0.0

 Autoplay plugin
 ✖ page auto-advances after allotted flip speed and delay
   1) AssertionError: expected
   'http://127.0.0.1:8000/BookReaderDemo/demo-autoplay.html#page/n1/mode/2up'
      to match /page\/n3/
      Browser: Chrome 83.0.4103.61 / Linux 0.0
          8 |
          9 |fixture `Autoplay plugin`.page `${BASE_URL}demo-autoplay.html`;
         10 |
         11 |test('page auto-advances after allotted flip speed and delay',
      async t => {
         12 |  await t.wait(FLIP_SPEED + FLIP_DELAY);
       > 13 |  await t.expect(getLocationHref()).match(/page\/n3/);
         14 |});
         15 |
         at <anonymous>
      (/home/travis/build/internetarchive/bookreader/tests/e2e/autoplay.test.js:13:37)
```

### Relevant files
https://github.com/internetarchive/bookreader/blob/70d1bbc547de37f6b4c30b77a3e410e311f5ed5d/tests/e2e/autoplay.test.js#L12

--- a/tests/e2e/autoplay.test.js
+++ b/tests/e2e/autoplay.test.js
@@ -4,7 +4,6 @@ const { BASE_URL } = process.env;
 const getLocationHref = ClientFunction(() => window.location.href.toString());
 const FLIP_SPEED = 1000;
 const FIRST_PAGE_DELAY = 1500;
-const FLIP_DELAY = 2000;
 
 fixture `Autoplay plugin`.page `${BASE_URL}demo-autoplay.html`;
 

--- a/tests/e2e/autoplay.test.js
+++ b/tests/e2e/autoplay.test.js
@@ -3,11 +3,12 @@ import { ClientFunction } from 'testcafe';
 const { BASE_URL } = process.env;
 const getLocationHref = ClientFunction(() => window.location.href.toString());
 const FLIP_SPEED = 1000;
+const FIRST_PAGE_DELAY = 1500;
 const FLIP_DELAY = 2000;
 
 fixture `Autoplay plugin`.page `${BASE_URL}demo-autoplay.html`;
 
 test('page auto-advances after allotted flip speed and delay', async t => {
-  await t.wait(FLIP_SPEED + FLIP_DELAY);
+  await t.wait(2 * FLIP_SPEED + FIRST_PAGE_DELAY);
   await t.expect(getLocationHref()).match(/page\/n3/);
 });

--- a/tests/e2e/autoplay.test.js
+++ b/tests/e2e/autoplay.test.js
@@ -3,7 +3,7 @@ import { ClientFunction } from 'testcafe';
 const { BASE_URL } = process.env;
 const getLocationHref = ClientFunction(() => window.location.href.toString());
 const FLIP_SPEED = 1000;
-const FIRST_PAGE_DELAY = 1500;
+const FIRST_PAGE_DELAY = 2000;
 
 fixture `Autoplay plugin`.page `${BASE_URL}demo-autoplay.html`;
 

